service/storage_service.cc

```diff
@@ -1411,11 +1463,16 @@ class topology_coordinator {
                 co_await handle_global_request(std::move(guard));
                 co_return true;
             }
+
+            if (auto feats = _topo_sm._topology.calculate_not_yet_enabled_features(); !feats.empty()) {
```

Comment:
I wonder if this should have more priority than user requests (i.e. put it before `handle_global_request`)

Reply:
I'm not sure either, but I'm not also sure if it matters. Do you have some concerns in particular?

Reply:
It probably doesn't matter